### DISCUSSION TITLE
API Access to Organization Repository Defaults

### DISCUSSION THREAD

**Discussion by clok** [github/rest-api-description#329] [Ideas]
I would like to build automation around managing GitHub Labels within an Organization. I have made good progress on this with the [ghlabels](https://github.com/clok/ghlabels) CLI tool that allows for a configuration to be passed in and used to sync across repos within a User or Org.

A desire I have is to interact with the Repository Defaults found within an Organization (see: `https://github.com/organizations/<ORG>/settings/repository-defaults`). For example, @GoodwayGroup we have 25 labels that we use across ~200 repositories. Syncing these Labels has been very helpful in maintaining a scalable SDLC as we group.

<img width="761" alt="image" src="https://user-images.githubusercontent.com/1429775/118183561-dcf4df80-b3ff-11eb-8522-781cd674f81a.png">

According to the available REST API documentation, I cannot find a way to interact with the Organization Repository Defaults. I am not sure where to make a feature request, so I thought here may be a good place to start.

Thanks!

**Comment by gr2m** [comment]
The GitHub Ecosystem forum at https://github.community/c/github-ecosystem/37 might be a good place to discuss your use case. It might be possible to access organization default labels using GraphQL, but I'm not sure

**Reply by clok** [reply]
Right on, opened a topic there as well: https://github.community/t/api-access-to-organization-repository-defaults/179844

**Reply by DamienCassou** [reply]
@clok: The link is broken.
